frontend/src/pages/Graphing/DashboardOverview.tsx

```diff
@@ -190,6 +194,106 @@ export default function DashboardOverview() {
 	)
 }
 
+const DashboardRow = ({
+	idx,
```

Comment:
Is this needed? I believe the `<DashboardRow key={row.id}...>` should suffice for the mapping key